packages/ncids-css/scss/components/nci-header/internal/_print.scss

```diff
@@ -0,0 +1,11 @@
+@media print {
+	.nci-header-nav,
+	.nci-header--extended .nci-header__navbar .nci-header-nav__secondary {
+		display:none;
```

Comment:
looks like this needs to go through linter and be formatted.  usually a space `display: none`